[34m[1mwandb[39m[22m: [33mWARNING[39m Calling wandb.run.save without any arguments is deprecated.Changes to attributes are automatically persisted.
I0207 16:52:30.836952 140120320071552 run.py:180] Using CLRS21 spec: {'train': {'num_samples': 1000, 'length': 16, 'seed': 11}, 'val': {'num_samples': 32, 'length': 16, 'seed': 21}, 'test': {'num_samples': 32, 'length': 64, 'seed': 31}}
I0207 16:52:34.292965 140120320071552 xla_bridge.py:231] Unable to initialize backend 'tpu_driver': NOT_FOUND: Unable to find driver in registry given worker:
2022-02-07 16:52:34.304806: E external/org_tensorflow/tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
I0207 16:52:34.305056 140120320071552 xla_bridge.py:231] Unable to initialize backend 'gpu': FAILED_PRECONDITION: No visible GPU devices.
I0207 16:52:34.305596 140120320071552 xla_bridge.py:231] Unable to initialize backend 'tpu': INVALID_ARGUMENT: TpuPlatform is not available.
W0207 16:52:34.305844 140120320071552 xla_bridge.py:235] No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
manual >>  True
adj shape >>  (32, 20, 20)
manual >>  True
adj shape >>  (32, 20, 20)
manual >>  True
adj shape >>  (32, 20, 20)
manual >>  True
adj shape >>  (32, 20, 20)
manual >>  True
adj shape >>  (32, 20, 20)
manual >>  True
adj shape >>  (32, 20, 20)
manual >>  True
adj shape >>  (32, 20, 20)
I0207 16:53:29.705042 140120320071552 run.py:261] Compiled feedback step in 55.415805 s.
Traceback (most recent call last):
  File "/usr/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/content/drive/MyDrive/Final Project/new_clrs/clrs/examples/run.py", line 411, in <module>
    app.run(main)
  File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 312, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.7/dist-packages/absl/app.py", line 258, in _run_main
    sys.exit(main(argv))
  File "/content/drive/MyDrive/Final Project/new_clrs/clrs/examples/run.py", line 273, in main
    split='train'
  File "/content/drive/MyDrive/Final Project/new_clrs/clrs/examples/run.py", line 240, in evaluate
    accuracy, shift, final = compute_average_metric(feedback, aux, 'advance_s', options)
  File "/content/drive/MyDrive/Final Project/new_clrs/clrs/examples/run.py", line 65, in compute_average_metric
    split, hints_to_log, step, available_splits = options['split'], options['hints'], options['step'], options['available_hints']
KeyError: 'available_hints'